IntType.h

```diff
@@ -1,6 +1,9 @@
 #pragma once
 #include <iostream>
```

Comment:
don't include what you don't use.  you aren't using iostream in this header file.
- [ ] revise